community-contributions/miscellaneous_reports/Increase Validator Set Research.md

```diff
@@ -0,0 +1,91 @@
+This report has been generated in response to Community Bounty 6 (#71).
+
+Task: Research the costs and parameter changes associated with increasing the Validator count substantially.
+
+# 1. Produce a report estimating the costs, for the validators themselves and other tJOY holders, associated with a larger validator set.
+## What is a good estimate of the (current) running costs for a validators? This assumes running a VPS from some of the bigger actors around.
+
+The costs for a VPS to run a reliable and stable validator node range from 5 to 15 $ / month depending on location and hosting provider.
+Hardware demands are around 40 to 50% constant CPU load for a 2.5 GHz core and at least 19 GB of disk space.
+
+## What is the weekly "cost" for the platform (in terms of tJOY and USD) the last 4 weeks?
+
+The total reward grew constantly from 18426 two weeks ago up to 20875 JOY per hour which is related to increasing amount staked.
+
+Current costs amount to 157$ per week (3.4 M tJOY).
+
+ time | payout JOY | payout $ | payout $ / validator
+-- | -- | -- | --
+per hour |    20.7 K  |       0.93  | 0.02
+per day |     496.8 K  | 45  | 0.15
+per week |    3.4776 M | 157  | 3.7
+per month |   13.9 M | 628 | 15
+
+## How much of the returns are not cashed out?
+
+I ran a script to compare validator stakes at present and 50 eras before:
```

Comment:
This is probably a good way to estimate, but it can be looked up with chain state queries!